sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.students.controller.getOfferingSummaryDetails.e2e-spec.ts

```diff
@@ -0,0 +1,183 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  FakeStudentUsersTypes,
+  getStudentToken,
+  getStudentByFakeStudentUserType,
+} from "../../../../testHelpers";
+import {
+  createE2EDataSources,
+  createFakeApplication,
+  createFakeEducationProgramOffering,
+  createFakeStudentAssessment,
+  saveFakeApplicationOfferingRequestChange,
+  createFakeUser,
+  E2EDataSources,
+} from "@sims/test-utils";
+import {
+  Application,
+  StudentAssessment,
+  Student,
+  EducationProgramOffering,
+} from "@sims/sims-db";
+import { Repository } from "typeorm";
+
+describe("EducationProgramOfferingStudentsController(e2e)-getOfferingSummaryDetails", () => {
+  let app: INestApplication;
+  let student: Student;
+  let db: E2EDataSources;
+  let applicationRepo: Repository<Application>;
+  let assessmentRepo: Repository<StudentAssessment>;
+
+  const deliveryMethod = (offering: EducationProgramOffering) => {
```

Comment:
I believe that usually, we keep the helper method at the end of the file. If it is something to be reused outside this file it an be moved to a different file/util.